packages/tabs/src/components/tabswidget.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+	<div>
+		<div
+			ref="tablist"
+			class="t-ui-tabswidget"
+			role="tablist"
+			@click="clickHandler"
+			@keydown="keydownHandler"
+		>
+			<slot name="tabs"></slot>
+		</div>
+		<div ref="panellist">
+			<slot name="panels"></slot>
```

Comment:
Slot here makes sense to provide a mechanism for implementors to add any kind of content.

Reply:
Is this actionable or rather thinking out loud? If the latter: Yes, that's what I was thinking.

Reply:
This was thinking out loud agreement :)